app/controllers/budgets_controller.rb

```diff
@@ -0,0 +1,47 @@
+class BudgetsController < ApplicationController
+  before_action :require_login
+
+  def index
+    @budgets = Budget.all
```

Comment:
This should be scoped to `current_user.budgets`
Otherwise, every user will be able to see every other user's budgets.
You should test that in a feature branch, that I can see only my budgets.

Reply:
Done 7807e6f